wpilibj/src/test/java/edu/wpi/first/wpilibj/MockSpeedController.java

```diff
@@ -8,12 +8,12 @@
 package edu.wpi.first.wpilibj;
 
 public class MockSpeedController implements SpeedController {
+  private double m_speed = 0.0;
+  private boolean m_isInverted = false;
```

Comment:
In java it could be better to not explicitly init because it adds approximately three additional bytecode instructions per field.